public/app/percona/check/CheckPanel.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { FC, useEffect, useMemo, useState } from 'react';
+// @ts-ignore
```

Comment:
Is there no other way, other than disabling ts? I suggest we create a tech-debt ticket to fix ts everywhere.

Reply:
Got rid of history and router here completely